src/network/networkd-link.c

```diff
@@ -1487,6 +1487,61 @@ static int link_set_group(Link *link) {
         return 0;
 }
 
+static int link_tx_rx_queues_hadler(sd_netlink *rtnl, sd_netlink_message *m, Link *link) {
+        int r;
+
+        assert(link);
+
+        if (IN_SET(link->state, LINK_STATE_FAILED, LINK_STATE_LINGER))
+                return 1;
+
+        r = sd_netlink_message_get_errno(m);
+        if (r < 0)
+                log_link_message_warning_errno(link, m, r, "Could not set transmit / receive queues for the interface");
+
+        return 1;
+}
+
+static int link_set_tx_rx_queues(Link *link) {
+        _cleanup_(sd_netlink_message_unrefp) sd_netlink_message *req = NULL;
+        int r;
+
+        assert(link);
+        assert(link->network);
+        assert(link->manager);
+        assert(link->manager->rtnl);
+
+        if (link->network->txqueues <= 0 && link->network->rxqueues <= 0)
```

Comment:
nit: these are unsigned variables, so they can never be < 0. Should be cosmetic only though.